src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/TransactionClosePaymentRetryQueueConsumer.kt

```diff
@@ -159,7 +156,7 @@ class TransactionClosePaymentRetryQueueConsumer(
       .onErrorResume(InvalidEventException::class.java) {
         logger.error("Invalid input event", it)
         writeEventToDeadLetterQueue(
-          checkPointer, payload, it, deadLetterQueueAsyncClient, deadLetterTTLSeconds)
+          checkPointer, payload, it, deadLetterTracedQueueAsyncClient, PARSING_EVENT_ERROR_CONTEXT)
```

Comment:
What do you think about moving `PARSING_EVENT_ERROR_CONTEXT` inside `DeadLetterTracedQueueAsyncClient.ErrorContext`'s companion object? (just for readability)